fcrepo-http-api/src/main/java/org/fcrepo/http/api/responses/StreamingBaseHtmlProvider.java

```diff
@@ -236,32 +207,18 @@ private Template getTemplate(final Model rdf, final Node subject,
                                   .findFirst();
 
         if (!template.isPresent()) {
-            LOGGER.trace("Attempting to discover mixin types of node for resource in question: {}", subject);
-            template = rdf.listObjectsOfProperty(createResource(subject.getURI()),
-                                                 createProperty(getRDFNamespaceForJcrNamespace(JCR_NAMESPACE) +
-                                                     "mixinTypes"))
-                          .toList().stream()
-                          .map(x -> x.asLiteral().getLexicalForm())
+            LOGGER.trace("Attempting to discover RDF types of node for resource in question: {}", subject);
+            template = rdf.listObjectsOfProperty(subjectURI, type).toList().stream()
```

Comment:
The other issue is that since we're still on Jena 2.13, the jena.util.iterator.Filter doesn't implement a functional interface. We could use the Guava `filter` and `transform` functions, but then we'd have a lot more nested code, which IMO is much less readable than this.